### PR TITLE
MicroProfile Metrics 4.0 - Jakartified

### PR DESCRIPTION
**Pull request by Channyboy**
#build
#17794

### CODE REVIEW

**Comment by Channyboy** [comment]
#build
#libby

**Comment by Channyboy** [comment]
#build 
#libby

File: dev/io.openliberty.microprofile.metrics.internal.4.0_fat_tck/publish/tckRunner/tck/pom.xml [hunk 1]
```diff
@@ -0,0 +1,269 @@
...
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>org.jboss.arquillian</groupId>
+                <artifactId>arquillian-bom</artifactId>
+                <version>${arquillian.version}</version>
+                <type>pom</type>
+                <scope>import</scope>
+            </dependency>
+            <dependency>
```
Comment (pgunapal) [inline dev/io.openliberty.microprofile.metrics.internal.4.0_fat_tck/publish/tckRunner/tck/pom.xml]: Currently, there isn't Jakarta version of `jboss-servlet-api_3.0_spec` dependency in maven, its best if we comment it out for now...

**Review by pgunapal — CHANGES_REQUESTED** [review]
Looks good, just one comment, why are we explicitly pushing the `dev/io.openliberty.microprofile.metrics.internal.4.0_fat_tck/publish/tckRunner/lib/hamcrest-all-1.3.jar` file into the repo, if we already have the following line:
`requiredLibs 'org.hamcrest:hamcrest-all:1.3'` in the build.gradle file of the TCK, so it automatically pulls the dependency during build time?

**Comment by Channyboy** [comment]
#build
#libby

Latest update squashed previous commits and includes changes that @pgunapal requested.

**Review by pgunapal — APPROVED** [review]
LTGM

**Pull request merged**